common/lib/xmodule/xmodule/tests/test_content.py

```diff
@@ -1,12 +1,20 @@
 """Tests for contents"""
+from StringIO import StringIO
```

Comment:
In general, you can use [cStringIO](https://docs.python.org/2/library/stringio.html#module-cStringIO) wherever you'd use `StringIO` and get a mild speed benefit.